admin-dev/themes/new-theme/js/pages/product/mixins/is-attribute-selected.ts

```diff
@@ -35,12 +37,12 @@ export default {
      *
      * @returns {boolean}
      */
-    isSelected(attribute, attributeGroup, attributeGroups) {
+    isSelected(attribute: Attribute, attributeGroup: AttributeGroup, attributeGroups: AttributeGroup): boolean {
```

Comment:
```suggestion
    isSelected(attribute: Attribute, attributeGroup: AttributeGroup, attributeGroups: AttributeGroup[]): boolean {
```

Reply:
This one is the most critical I think, this is what brings most of the errors in the combination generator